### PR TITLE
feat: adding missing docstrings for functions & classes

### PR DESCRIPTION
**Pull request by tina80lvl**
Added descriptions for classes' and functions' functionalities and arguments.

### CODE REVIEW

**Comment by tina80lvl** [comment]
@googlebot I consent.

**Comment by mf2199** [comment]
@googlebot I consent.

**Comment by c24t** [comment]
Checked manually that all committers have signed the CLA.

**Review by c24t — COMMENTED** [review]
A few comments, but this looks like a clear improvement.

There are many places where the docstrings don't render well in sphinx. See https://github.com/q-logic/python-spanner-django/pull/24 to preview the generated html. A good next step would be to clean up the docstrings for *all* the public-facing modules and check that they render nicely. Ultimately this should look a lot like python-spanner's API docs: https://googleapis.dev/python/spanner/latest/api-reference.html.

File: google/cloud/spanner_dbapi/cursor.py [hunk 1]
```diff
@@ -93,7 +96,11 @@ def description(self):
 
     @property
     def rowcount(self):
-        """The number of rows produced by the last `.execute()`."""
+        """The number of rows produced by the last `.execute()`.
```
[the remark this answers is not in the document]
Comment (c24t) [reply google/cloud/spanner_dbapi/cursor.py:99]: To change the default role so backticks render sanely: https://github.com/googleapis/python-spanner-django/pull/550.

**Review by c24t — APPROVED** [review]
LGTM, but note that these changes will have to get picked onto python-spanner now that https://github.com/googleapis/python-spanner/pull/160 is merged.

**Comment by mf2199** [comment]
@googlebot I consent.

**Comment by AVaksman** [comment]
The functionality was moved to https://github.com/googleapis/python-spanner repo

**Pull request closed without merging**